### ISSUE TITLE
GetAll not updating localMapStats (hz 3.7.3)

### ISSUE THREAD

**Issue by fredang** [hazelcast/hazelcast#9373]
Hi,

I did a small test where I call Get/GetAll/Put/PutAll multiple times on a distributed map and then output the local map stats.
The Get/Put/PutAll are showing properly in the local map stats.
However this is not the case for GetAll (seems to be excluded from the stats).

To reproduce the issue:
```
        HazelcastInstance instance = Hazelcast.newHazelcastInstance();

        IMap<String, Integer> map = instance.getMap("test");
        for(int i = 0 ; i < 100 ; i++) {
            Map<String, Integer> putMap = new HashMap<>();
            String key1 = "key1-" + i;
            String key2 = "key2-" + i;
            putMap.put("key1-" + i, i);
            putMap.put("key2-" + i , i);
            map.putAll(putMap);


            Set<String> keys = new HashSet<>();
            keys.add(key1);
            keys.add(key2);
            map.getAll(keys);
        }
        System.out.println(map.getLocalMapStats());

```

It outputs the following stats
```
LocalMapStatsImpl{lastAccessTime=1480968873116, lastUpdateTime=1480968873116, hits=200, 
numberOfOtherOperations=0, numberOfEvents=0, getCount=0, putCount=200, removeCount=0,
totalGetLatencies=0, totalPutLatencies=120, totalRemoveLatencies=0, ownedEntryCount=200,
backupEntryCount=0, backupCount=1, ownedEntryMemoryCost=31780, backupEntryMemoryCost=0,
creationTime=1480968872891, lockedEntryCount=0, dirtyEntryCount=0, heapCost=31780}
```

**Comment by jerrinot** [comment]
fixed by #9387

**Comment by jerrinot** [comment]
:clap: for @fredang !
